packages/wallets/src/components/DatePicker/__tests__/DatePicker.spec.tsx

```diff
@@ -14,7 +14,16 @@ jest.mock('../../FlowProvider', () => ({
     useFlow: jest.fn(() => mockUseFlow),
 }));
 
-describe('DatePicker Component', () => {
+describe('DatePicker', () => {
+    beforeAll(() => {
+        jest.useFakeTimers('modern');
+        jest.setSystemTime(new Date('2024-01-15').getTime());
+    });
+
+    afterAll(() => {
+        jest.useRealTimers();
+    });
```

Comment:
Updated DatePicker unit test to use fake timers to remove dependency on real time/date and avoid time/date complications